cmd/otelcontribcol/Dockerfile

```diff
@@ -1,12 +1,19 @@
-FROM alpine:latest as certs
+FROM alpine:latest as prep 
 RUN apk --update add ca-certificates
 
+ARG USER_UID=10001
+
+# required by podmanreceiver 
```

Comment:
Please add in the comment why these are required by podmanreceiver.

Reply:
expanded the comment